manifests/a/Amazon/Chime/4.35.10012.1/Amazon.Chime.yaml

```
@@ -1,5 +1,5 @@
 PackageIdentifier: Amazon.Chime
-PackageVersion: 4.35.10012
+PackageVersion: 4.35.10012.1
 PackageName: Amazon Chime
 Publisher: Amazon
```

```suggestion
Publisher: Amazon.com, Inc.
```
Make sure to copy and paste whatever you see in Windows Sandbox from Control Panel onto here.